### DISCUSSION TITLE
HERO10 BLACK Labs Firmware v1.60.70 No Longer Available For Download

### DISCUSSION THREAD

**Discussion by asylthe** [gopro/labs#863] [General]
The other day I downloaded and installed the HERO10 v1.60.70 labs firmware on my camera.  I see that it's no longer available for download.  Why was it pulled from the site?  Should I not be running that firmware?  I'm assuming it was pulled down for a reason.

**Comment by dnewman-gpsw** [comment]
It was broken in some languages causing some menu display issues, we are fixing this now.  If you don't see any issue, your are language didn't have the problem, you will not need to roll back.  A new version is expected very soon.

**Reply by asylthe** [reply]
I appreciate the quick response!  I have the language on my GoPro set to English, and haven't noticed any issues.  Was English one of the affected languages?

**Reply by dnewman-gpsw** [reply]
I don't think so, I didn't see anything.
